docs/src/main/sphinx/language/types.rst

```diff
@@ -168,8 +168,9 @@ Example: ``TIME '01:02:03.456'``
 
 Time of day (hour, minute, second, millisecond) with a time zone.
 Values of this type are rendered using the time zone from the value.
+Time zones are expressed as the numeric UTC offset value.
```

Comment:
Lets change this to our default example setup that has the sql query and the output as comment so you can actually cut and paste to try

```
Time zones are expressed as the numeric UTC offset value::

    SELECT TIME '01:02:03.456 -08.00';
    == 1:02:03.456-08:00

Reply:
@mosabua Is the query meant to have the : instead of the . ? Or just the output?

Reply:
Both the query and the output.